### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mossbeard**
Is code ok, please create a nuget

### CODE REVIEW

File: src/Q2gHelperQlik.csproj [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <CreateProperty Value="0.0.1-notgitversion" Condition="'$(GitVersion_NuGetVersion)' == ''">
+      <Output TaskParameter="Value" PropertyName="Version" />
+    </CreateProperty>
+    <PropertyGroup Condition="'$(GitVersion_NuGetVersion)' == ''">
+      <DefineConstants>$(DefineConstants);MISSINGGIT;</DefineConstants>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="FixOnGitVersionInfoTaskError" BeforeTargets="GetAssemblyVersion">
+    <CreateProperty Value="$(GitVersion_InformationalVersion)">
```
Comment (konne) [inline src/Q2gHelperQlik.csproj]: @mossbeard please move this block one block up

File: src/Q2gHelperQlik.csproj [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <Output TaskParameter="Value" PropertyName="FullVersionInfo" />
+    </CreateProperty>
+    <PropertyGroup Condition="'$(GitVersion_InformationalVersion)' == ''">
+      <FullVersionInfo>0.0.1-notgitversion</FullVersionInfo>
+    </PropertyGroup>
+  </Target>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
```
Comment (konne) [inline src/Q2gHelperQlik.csproj]: please add an "or" in the condition

Comment (konne) [reply src/Q2gHelperQlik.csproj]: or check $(MyProperty.Substring(0, 4) = net4

Comment (konne) [reply src/Q2gHelperQlik.csproj]: in the build file take one position to combine all check and set a new
for example: <DefineConstants>$(DefineConstants);CLASSICNET;</DefineConstants>
And us this the the code with.
#if CLASSICNET

#endif

**Review by konne — CHANGES_REQUESTED** [review]
@mossbeard please see comments

**Comment by mossbeard** [comment]
@konne  
I have chnage the project file now. In this case, I still have old states in the .cs files noticed that I have fixed with the same.

**Review by konne — APPROVED** [review]
@mossbeard please merge

**Pull request merged**